core/src/test/java/org/elasticsearch/search/aggregations/metrics/TDigestPercentileRanksIT.java

```diff
@@ -389,14 +411,25 @@ public void testScriptMultiValued() throws Exception {
     public void testScriptMultiValuedWithParams() throws Exception {
         Map<String, Object> params = new HashMap<>();
         params.put("dec", 1);
+
+        // Equivalent to:
+        //
+        // List values = doc['values'].values;
+        // double[] res = new double[values.size()];
+        // for (int i = 0; i < res.length; i++) {
+        //      res[i] = values.get(i) - dec;
+        // };
+        // return res;
+        Script script = new Script("decrement all values", ScriptType.INLINE, AggregationTestScriptsPlugin.NAME, params);
```

Comment:
I wonder if it is worth adding `public static final Script DECREMENT_ALL_VALUES` to the plugin and referencing it rather than building this in all the tests. Just from a clarity standpoint.